data/routers/project-routers.js

```diff
@@ -0,0 +1,159 @@
+const express = require("express");
+const router = express.Router();
+const projects = require("../helpers/projectModel");
+
+router.get("/api/projects", (req, res) => {
+  projects
+    .get(req.body.id)
+    .then(post => {
+      if (post === null) {
+        res.status(404).json({ message: "There is no project with this id" });
+      } else {
+        res.status(200).json(post);
+      }
+    })
+    .catch(error => {
+      res
+        .status(500)
+        .json({ message: "An error occurred collecting the data" });
+    });
+});
+
+router.get("api/projects/:id/actions", (req, res) => {
+  const { id } = re.params;
+  projects
+    .get(req.params.id)
+    .then(post => {
+      if (post === null) {
+        res.status(404).json({ message: "There isn't a project with this id" });
+      } else {
+        projects
+          .getProjectActions(req.params.id)
+          .then(actions => {
+            if (actions.length === 0) {
+              res
+                .status(200)
+                .json({ message: "There isn't a project with this id" });
+            } else {
+              res.status(200).json(actions);
+            }
+          })
+          .catch(error => {
+            res
+              .status(500)
+              .json({ message: "There was an error collecting action data" });
+          });
+      }
+    })
+    .catch(error => {
+      res
+        .status(500)
+        .json({ message: "There was an error collecting action data" });
+    });
+});
+
+router.post("/api/projects", (req, res) => {
+  const { name, description } = req.body;
+  if (req.body.name && req.body.description) {
+    projects
+      .insert(req.body)
+      .then(post => {
+        res
+          .status(201)
+          .json({ message: "The Post has bee successfully created." });
+      })
+      .catch(error => {
+        res.status(500).json({ error: error.message });
+      });
+  } else {
+    res.status(400).json({
+      message: "Please provide a name and a description for your new post"
+    });
+  }
+});
+
+router.delete("/api/projects/:id", (req, res) => {
+  const { id } = req.params;
+  projects
+    .remove(req.params.id)
+    .then(code => {
+      if (code === undefined) {
```

Comment:
The result of deletion is usually `1` which is the number of rows on the database affected by the delete operation.
SO in this case, on successful deletion, you should compare with 1